frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/admin/ImportExportPortlet.java

```diff
@@ -326,4 +339,22 @@ public void updateReleaseLinks(ActionRequest request, ActionResponse response) t
 
         renderRequestSummary(request, response, requestSummary);
     }
+    public void backUpLicenses(ResourceRequest request, ResourceResponse response) throws IOException, TException {
```

Comment:
As the same method is already present in `LicenseAdminPortlet`. May be we should consider to move it to a common place like `Sw360Portlet`